spec/tests/yaml/swiftnav/sbp/test_piksi.yaml

```diff
@@ -0,0 +1,561 @@
+---
+description: Unit tests for swiftnav.sbp.piksi v0.23.
+generated_on: 2015-03-24 11:31:59.676598
+package: sbp.piksi
+tests:
+
+- msg:
+    fields: null
```

Comment:
And If there's something we can hook into there with tracing levels from the firmware, all the better.